### PR TITLE
Update config to use kubectl-gadget v0.12.1

### PR DESCRIPTION
**Pull request by blanquicet**
This PR only updates the config to use the latest version released for Inspektor Gadget: [v0.12.1](https://github.com/inspektor-gadget/inspektor-gadget/releases/tag/v0.12.1).

cc @Tatsinnit

### CODE REVIEW

**Review by Tatsinnit — APPROVED** [review]
❤️☕️🙏 hiya Jose! thank you so much for this, looks good.

**Review by peterbom — APPROVED** [review]

**Comment by blanquicet** [comment]
I don't have write access to this repo so I can't merge it. It needs to be one of you 🙂

**Pull request merged**
